dbos/_db_wizard.py

```diff
@@ -46,25 +49,43 @@ def db_wizard(config: "ConfigFile", config_file_path: str) -> "ConfigFile":
         )
     db_config = config["database"]
 
-    # Read the config file and check if the database hostname/port/username are set. If so, skip the wizard.
-    with open(config_file_path, "r") as file:
-        content = file.read()
-        local_config = yaml.safe_load(content)
-        if "database" not in local_config:
-            local_config["database"] = {}
-        local_config = cast("ConfigFile", local_config)
+    # Read the config file and check if the database hostname/port/username are set
```

Comment:
Just remove this check entirely, with the new approach it's not necessary (and overcomplicated, accessing the yaml file directly from an unusual place)